PowerEditor/installer/nativeLang/polish.xml

```diff
@@ -486,31 +552,33 @@
                     <Item id="25017" name="Linia komentarza"/>
                     <Item id="25026" name="Operator 1"/>
                     <Item id="25027" name="Operator 2"/>
-                    <Item id="25028" name="Liczby"/>
+                    <Item id="25028" name="Numery"/>
```

Comment:
IMO previous translation for this was better.

Reply:
Considering the context, I agree.